SRC/NBASIC/nbas111e.txt

```diff
@@ -290,54 +289,54 @@ S+6: VRAM, addresses &H04000-&H07FFF (not available with 64K VRAM)
 S+7: VRAM, addresses &H00000-&H03FFF (not available with 64K VRAM)
 
 S+8 to 254: Not available (if S+8<255)
-255: BASIC main RAM (&H8000-&HFFFF)
+255: MSX-BASIC main RAM (&H8000-&HFFFF)
 
 
 3.5. ERRORS
 
-All the functions which perform RAM and/or VRAM access returns the error code 
--1 if any non existing RAM segment or VRAM address (addresses above &HFFFF in 
-computers with 64K VRAM) is specified in any input parameter, or if any VRAM 
-segment or the 255 segment is specified and the function supports only normal 
-RAM segments.
+All the functions which perform RAM and/or VRAM access return the error code   
+-1 if any nonexistent RAM segment or VRAM address (addresses above &HFFFF in 
+computers with 64K VRAM) is passed to a function, or if any VRAM segment or
+segment 255 is specified as input parameter for a function that supports normal
+RAM segments only.
 
 
 4. DISK ACCESS
 
-NestorBASIC has functions for disk files management and for the use of other 
+NestorBASIC includes functions that work with disk files and use other        
 MSX-DOS capabilities:
 
 * Create/delete/rename/search files.
 * Read/write files to/from RAM or VRAM.
 * Under DOS 2, move files and get/set file attributes.
 * Read/write disk sectors to/from RAM or VRAM.
-* Get/set the default drive/directory.
+* Get/set the current working drive/directory.
 * Obtain the disk size and free space.
-* Under DOS 2, get size/set the RAM disk.
+* Under DOS 2, get size of/create the RAM disk.
 
 
 4.1. THE ARRAY F$
 
-For passing filenames and pathnames to these functions, a string array named 
-F$ is used. The functions for search, rename and move files, and for parse a 
-pathname need two strings are needed, so F$ must be defined with two elements 
-[DIM F$(1)]. The rest of disk access functions need only one string; 
-therefore, if none of the four functions mentioned above are needed, it is 
-recommended to define the array with only one element [DIM F$(0)], due to the 
-string storage method of TurboBASIC: each string uses 256 bytes in RAM, 
-regardless of its real length.
+For passing filenames and pathnames to these functions, a string array named    
+F$ is used. Functions for search, renaming and moving files, and for parsing  
+pathname require two strings, so F$ of two elements has to be declared
+[DIM F$(1)]. Rest of the functions require just one string, therefore, if none
+of that four functions mentioned above are needed, you can define the array
+with only one element [DIM F$(0)]. It is especially recommended in case of

```

Comment:
"It is" -> "This is", to make it clear that we are talking about declaring the array with only one element.